src/Components/ServiceSelectionScene/ServiceSelectionScene.tsx

```diff
@@ -677,7 +730,25 @@ export class ServiceSelectionScene extends SceneObjectBase<ServiceSelectionScene
     }
   }
 
-  private updateBody() {
+  private getGridItems(): SceneCSSGridItem[] {
+    return this.state.body.state.children as SceneCSSGridItem[];
+  }
+
+  private getVizPanel(child: SceneCSSGridItem) {
+    return child.state.body as VizPanel | undefined;
```

Comment:
```suggestion
    return child.state.body instanceof VizPanel ? child.state.body  : undefined;
```

Reply:
Avoids the type assertion too. Good catch and fixed